aep/general/0300/aep.md.j2

```diff
@@ -0,0 +1,98 @@
+# AEP Editions
+
+This AEP describes how the AEP specification is versioned.
+
+## Edition based versioning
+
+AEPs will be published in editions, similar but not identical to specifications
+such as those for the C++ or Rust programming language:
+
+- The specification will not change for a non-preview edition after it is
+  published.
+- The naming of the edition will be based on the year in which it is published
+  (e.g. aep-2025).
+- Upon publishing a non-preview edition, the current prevision will be removed,
+  and the next preview edition will be created (e.g. aep-2027-preview). This
+  preview edition will accumulate changes until the next edition is published.
```

Comment:
```suggestion
  preview edition will accumulate changes until it is promoted to stable.
```

WDYT?  As written, it sounds a little like the preview edition isn't that closely related to the next edition.